cms/templates/cms/home_page.html

```diff
@@ -96,14 +96,21 @@
                       {% else %}
                       <div class="widget-metas"><span class="meta-courses">{{ program.course_set.count }} courses</span></div>
                       {% endif %}
-                      <div class="widget-rating">
-                        <div class="rating rating-lg blue-600" data-score="4" data-read-only="true" data-class="blue-600" data-plugin="rating"></div>
-                      </div>
                     </div>
                   </div>
                 </div>
               </li>
               {% endfor %}
+              <li class="col-sm-6">
+                <div class="media-item">
+                  <div class="title-coming-soon text-center">Many more programs coming soon...</div>
+                  <img class="img-responsive center-block" src="static/images/academia.png"
+                       alt="image for program comming soon"/>
+                   <div class="description-coming-soon">
+                        Lorem ipsum dolor sit amet, pro tantas causae imperdiet ei, sea no elitr deterruisset. Qui dolor laboramus voluptatum ut. Ad his amet timeam eripuit. Et utinam primis verterem nec.
```

Comment:
@pdpinch can you send me the text that i should show on description for placeholder card?